src/components/test-slot/candidate-link/candidate-link.ts

```diff
@@ -12,7 +12,7 @@ import { CANDIDATE_DETAILS_PAGE, FAKE_CANDIDATE_DETAILS_PAGE } from '../../../pa
 })
 export class CandidateLinkComponent {
   @Input()
-  slotId: number | string;
+  slot: any;
```

Comment:
can we strongly type this instead of any?

Reply:
We could but then we have to delete a unit test because the `slotId` is a string for practice test and in the schema is a number.

Reply:
I'll resolve this conversation and merge the PR. But we can have a chat about this and come up with a way of getting that strongly typed.